lerobot/configs/default.yaml

```
@@ -57,3 +57,29 @@ wandb:
   disable_artifact: false
   project: lerobot
   notes: ""
+
+image_transform:
+  enable: true
+  # Maximum number of transforms to apply
```

Maybe this adds more clarity? But I'm also not clear on "maximum" here. But I'm waiting to get clarification on the `p` parameter that I asked about above.

```suggestion
  # A subset of these transforms will be applied for each batch. This is the maximum size of that subset.
```

---

Fixed